classes/Xdmod/Version.php

```diff
@@ -35,14 +35,7 @@ class Version
      */
     protected static $databaseHelpers = array();
 
-    /**
-     * Name of the schema version history table.
-     *
-     * @var string
-     */
-    protected static $versionTableName = 'schema_version_history';
-
-    /**
+      /**
```

Comment:
```suggestion
    /**
```
Indentation shouldn't change here.